editor/editor_file_dialog.cpp

```diff
@@ -198,8 +198,18 @@ void EditorFileDialog::update_dir() {
 
 	dir->set_text(dir_access->get_current_dir());
 
-	// Disable "Open" button only when we in selecting file(s) mode or open dir mode.
+	// Disable "Open" button only when selecting file(s) mode.
 	get_ok()->set_disabled(_is_open_should_be_disabled());
+	switch (mode) {
+
+		case MODE_OPEN_FILE:
+		case MODE_OPEN_FILES:
+			get_ok()->set_text(RTR("Open"));
```

Comment:
`RTR` is only for runtime (`scene`) stuff, for editor-specific stuff you can use `TTR`.